packages/unchained-client/src/evm/ethereum/parser/__tests__/mockData/tokens.ts

```diff
@@ -43,10 +43,10 @@ export const tribeToken: Token = {
 }
 
 export const maticToken: Token = {
-  contract: '0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0',
+  contract: '0x455e53CBB86018Ac2B8092FdCd39d8444aFFC3F6',
```

Comment:
✅

Reply:
Actually, we should probably keep the previous flavour here, *or* update to POL, see https://polygontechnology.notion.site/MATIC-POL-Migration-3367ffa954d743b88200cb25cd9f504c